src/mlpack/methods/ann/rbm.hpp

```diff
@@ -0,0 +1,183 @@
+/**
+ * @file rbm.hpp
+ * @author Kris Singh
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_RBM_HPP
+#define MLPACK_METHODS_ANN_RBM_HPP
+
+#include <mlpack/core.hpp>
+#include <mlpack/prereqs.hpp>
```

Comment:
``core.hpp`` already includes ``prereqs.hpp``, so we can remove the header here.